uaclient/contract.py

```diff
@@ -115,8 +115,8 @@ def request_resource_machine_access(
 
         @param machine_token: The authentication token needed to talk to
             this contract service endpoint.
-        @param resource: Entitlement name. One of: livepatch, esm, fips or
-            fips-updates.
+        @param resource: Entitlement name. One of: livepatch, esm-infra, fips
```

Comment:
Is this list actually accurate? As they're dynamic, should we just drop this instead of updating it?

Reply:
Good suggestion. Dropped.